nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -240,13 +240,7 @@ let
 
       server_tokens ${if cfg.serverTokens then "on" else "off"};
 
-      ${optionalString cfg.proxyCache.enable ''
-        proxy_cache_path /var/cache/nginx keys_zone=${cfg.proxyCache.keysZoneName}:${cfg.proxyCache.keysZoneSize}
-                                          levels=${cfg.proxyCache.levels}
-                                          use_temp_path=${if cfg.proxyCache.useTempPath then "on" else "off"}
-                                          inactive=${cfg.proxyCache.inactive}
-                                          max_size=${cfg.proxyCache.maxSize};
-      ''}
+      ${mkProxyCache}
```

Comment:
`mkProxyCache` sounds like a function name, but this is a string here.
Look how `mkLocation` are proper functions.
You should do the same here and call `mkProxyCache something` here.